ERCS/erc-x.md

```diff
@@ -0,0 +1,184 @@
+---
+eip: x
```

Comment:
```suggestion
eip: 7741
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.